accepted/pl/PSR-0.md

```diff
@@ -0,0 +1,74 @@
+Standardy autoloadingu
+======================
+
+> **Zdeprecjonowany** - Od dnia 2014-10-21 standard PSR-0 jest standardem zdeprecjonowanym. [PSR-4] jest od teraz rekomendowany jako alternatywa.
+
+[PSR-4]: http://www.php-fig.org/psr/psr-4/
+
+Poniższy dokument opisuje obowiązkowe wymagania, które należy stosować w procesie standaryzacji autoloadingu klas.
+
+Wymagania
+---------
+
+* W pełni poprawna przestrzeń nazw (namespace) i klasa muszą posiadać następującą strukturę 
+`\<Nazwa Vendora>\(<Namespace>\)*<Nazwa klasy>`
+* Każdy namespace musi posiadać namespace bazowy ("Nazwa Vendora").
+* Każdy namespace może posiadać tyle podnamespace'ów ile chce.
```

Comment:
namespace - `przestrzeń nazw` to tłumaczenie obowiązuje powszechnie w literaturze.